dbms/src/Server/Server.cpp

```diff
@@ -1187,34 +1191,37 @@ int Server::main(const std::vector<std::string> & /*args*/)
     }
     LOG_DEBUG(log, "Load metadata done.");
 
-    /// Then, sync schemas with TiDB, and initialize schema sync service.
-    for (int i = 0; i < 60; i++) // retry for 3 mins
+    if (!global_context->isDisaggregatedComputeMode())
```

Comment:
Looks like we also need to disable 
- `SegmentReaderPoolManager` in L1164.
- DeltaIndexManager

And will `proxy_conf.is_proxy_runnable` be false for compute mode?

Reply:
1. We disable `schema syncer` because don't want network connection between compute node and tikv. Other storage components doesn't need to disable.
2. Milestone 1 still need `proxy` to help connecto with PD. Milestone 2 will disable `proxy`.